packages/dataviews/src/components/dataviews/stories/fixtures.tsx

```diff
@@ -616,8 +617,9 @@ export const actions: Action< SpaceObject >[] = [
 
 export const fields: Field< SpaceObject >[] = [
 	{
-		label: 'Image',
 		id: 'image',
+		type: 'text', // TODO: add media/image type
```

Comment:
I find that `media` is the only basic type missing. We already have a couple of cases in our codebase where this could be used (featured media, preview).

[Currently](https://github.com/WordPress/gutenberg/blob/ee5e8be0505ed070b84bab23df1ce62dab5c8bd4/packages/dataviews/src/field-types/index.tsx#L28), the `type` only provides `sort`, `isValid` and `Edit` functions, but nothing stops us from providing more (and we should). For example, `type: media` fields could provide `enableSorting: false` by default.